.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+before_install:
+  - sudo add-apt-repository --yes ppa:kalakris/cmake
```

Comment:
nm, found the answers here:

https://gist.github.com/winterz/10cc2741d466cbd8ff21
https://github.com/travis-ci/travis-ci/issues/2030

might be good to add a comment linking these open travis issues